sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/OpenTelemetryBuilderExtensions.cs

```diff
@@ -122,7 +124,11 @@ public static OpenTelemetryBuilder UseAzureMonitor(this OpenTelemetryBuilder bui
                                 return true;
                             })
                             .AddProcessor<ProfilingSessionTraceProcessor>()
-                            //.AddLiveMetrics() TODO: Enable LiveMetrics
+                            .AddProcessor(sp =>
+                            {
+                                var manager = sp.GetRequiredService<Manager>();
+                                return new LiveMetricsActivityProcessor(manager);
+                            })
```

Comment:
this should be encapsulated by the `AzureMonitorOptions.EnableLiveMetrics` check.

https://github.com/Azure/azure-sdk-for-net/blob/5d3eede70ac10612e1d2360ff9208cba695cc9b0/sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/AzureMonitorOptions.cs#L47

Reply:
We will consider this in a follow up after this release. Currently we have this logic in Manager which cover it up.